src/main/java/com/geronimo/controller/HomeController.java

```diff
@@ -8,9 +8,14 @@
 @RestController
 public class HomeController {
 
-    @PreAuthorize("permitAll")
     @GetMapping("/")
```

Comment:
This page will be secured by web security, each URL require user to be authenticated